components/brave_ads/browser/ads_service.h

```diff
@@ -16,6 +16,8 @@
 
 namespace brave_ads {
 
+using IsSupportedRegionCallback = base::Callback<void(bool)>;
```

Comment:
`base::Callback` is deprecated. Currently it is an alias for `base::RepeatingCallback` but it is expected to be flipped to default to `base::OnceCallback`. I know there's tons of these laying around but we should specify now so we don't run into problems later when it does get flipped.

https://chromium.googlesource.com/chromium/src/+/master/base/callback.h#48

From looking at https://github.com/brave/brave-core/pull/1941/files#diff-c5c17b9ca47778443579990368dc025fR118, it looks like you want a `base::OnceCallback`

Reply:
Now using `base::OnceCallback`